docs/pages/includes/tls-certificate-setup.mdx

```diff
@@ -17,12 +17,14 @@ deployments, use your own private key and certificate.
     used for notifications (you can use any domain):
 
     ```code
-    $ DOMAIN=tele.example.com
-    $ EMAIL=user@example.com
-    $ teleport configure --acme --acme-email=${EMAIL?} --cluster-name=${DOMAIN?} | \
+    $ teleport configure --acme --acme-email=<Var name="user@example.com" description="Your email address to register with Let's Encrypt for TLS certificates" /> --cluster-name=<Var name="tele.example.com" description="The domain name of your Teleport cluster" /> | \
     sudo tee /etc/teleport.yaml > /dev/null
     ```
 
+    <Admonition type="tip">
+    Depending on your installation method you may need to run this command as `root` or with `sudo`.
```

Comment:
sudo is already included in the command above, so I find this tip a little confusing.